include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -2110,20 +2008,9 @@ inline DerivativeForm<1, dim, spacedim, Number>
 FEPointEvaluation<n_components, dim, spacedim, Number>::jacobian(
   const unsigned int point_index) const
 {
-  const auto &mapping_data =
-    mapping_info->get_mapping_data(current_cell_index, current_face_number);
-
-  const unsigned int n_lanes = internal::VectorizedArrayTrait<Number>::width;
-  DerivativeForm<1, dim, spacedim, Number> vectorized_jacobian;
-  for (unsigned int v = 0;
-       v < n_lanes && point_index * n_lanes + v < mapping_data.jacobians.size();
-       ++v)
-    for (unsigned int d = 0; d < dim; ++d)
-      for (unsigned int s = 0; s < spacedim; ++s)
-        internal::VectorizedArrayTrait<Number>::get(vectorized_jacobian[d][s],
-                                                    v) =
-          mapping_data.jacobians[point_index * n_lanes + v][d][s];
-  return vectorized_jacobian;
+  AssertIndexRange(point_index, n_q_points);
+  Assert(jacobian_ptr != nullptr, ExcMessage("jacobian_ptr is not set!"));
```

Comment:
The message does not really help. I think in the case of `FEEval` the message tells you what to do:

https://github.com/dealii/dealii/blob/aef0f7024129c7d25920d269ce39322165183c92/include/deal.II/matrix_free/fe_evaluation.h#L4522-L4524

Reply:
Are you suggesting declaring an own exception with a verbose message what to do or to improve the current messages in place?

Reply:
We have a verbose exception now which tells the user what to do!